src/main/java/core/basesyntax/database/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.database;
+
+import core.basesyntax.model.RequestBarvinok;
+import core.basesyntax.model.RequestVeselka;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<RequestVeselka> REQUEST_VESELKAS = new ArrayList<>();
```

Comment:
is it constant? write name of the variable using camelCase

Reply:
ok